TaskWithWritingInFile/TaskWithWritingInFileWay1/TaskWithWritingInFileWay1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <fstream> 
+
+int main()
+{
+	std::ofstream file;
+	char name[] = "";
+	std::cin >> name;
```

Comment:
You overflow the array name. What happens if user input is more than sizeof(name)=1? Please declare name as std::string in this example.